### DISCUSSION TITLE
I'm dropping the project

### DISCUSSION THREAD

**Discussion by delzear** [delzear/hass-formulaoneapi#74] [Announcements]
Hi guys, I will be dropping this project. I don't have time to manage it the way it should. So if somebody want to take over please contact me.

**Comment by brhahlen** [comment]
Can you give some more insight on what is needed to run this?

**Comment by brhahlen** [comment]
@delzear Could you let us know? I _might_ be willing to take over, just need to know what is involved in running this.

**Comment by delzear** [comment]
We are not allowed to query the Ergast API in a loop. So I had to build a cache server. Since the feeds were split my server (It is very weak, could not afford a VPS) has 5 times the requests and my other projects are suffering. Since I am a .Net dev, the cache server is built in .Net and is pretty simple.

**Reply by brhahlen** [reply]
Thanks!
Does the cache server run in Docker?
I have a home-server, but I try to run as much as possible in Docker, so that would definitely help.

How many requests are we talking about here?

**Comment by delzear** [comment]
Was not built this way but probably. I don't have the stats, But the Ergast API fails alot lately, causing my cache server to fail requests from the integration. Since I'm not a python dev and have loads of other projects, I don't have time to solve the problems. I might be convinced to keep the server if the integration was taken over by somebody else and the frequency was modified.

To calculate the stats, every install, 5 request every 60 seconds!

**Reply by brhahlen** [reply]
Hmm... I mean, I see that the default interval is set to 60 seconds, maybe we should set that to half an hour, or an hour maybe even. There's only so many times that the data needs to be accessed, especially on non-F1 weeks/weekends.

I'll create a fork and see if I can change this.

**Comment by brhahlen** [comment]
I just realized that the custom:formulaone-card works without this API, which is what I (and probably others) were using it for.
Which is why I will be uninstalling this API from HASS.
However, I do hope that my contributions, however small, will help you!
